### PR TITLE
Added release notes

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Based on https://help.stackrox.com/docs/release-notes/

Preview: 
- https://openshift-docs-git-release-notes-gnelson.vercel.app/openshift-acs/master/release_notes/3061-release-notes.html
- https://openshift-docs-git-release-notes-gnelson.vercel.app/openshift-acs/master/release_notes/3060-release-notes.html
- https://openshift-docs-git-release-notes-gnelson.vercel.app/openshift-acs/master/release_notes/3059-release-notes.html
- https://openshift-docs-git-release-notes-gnelson.vercel.app/openshift-acs/master/release_notes/3058-release-notes.html
- https://openshift-docs-git-release-notes-gnelson.vercel.app/openshift-acs/master/release_notes/3057-release-notes.html
- https://openshift-docs-git-release-notes-gnelson.vercel.app/openshift-acs/master/release_notes/3056-release-notes.html

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@bergerhoffer PTAL

Please note that I did not modify them a lot. We want to keep release notes for the last 6 releases.

File: _topic_map.yml [hunk 1]
```diff
@@ -29,6 +29,23 @@ Topics:
 - Name: Welcome
   File: index
 ---
+Name: Release Notes
```
Comment (bergerhoffer) [inline _topic_map.yml:32]: Release notes

File: release_notes/3056-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+:context: 3056-release-notes
+
+toc::[]
+
+[role="_abstract"]
+{product-title} version 3.0.56 includes feature enhancements, bug fixes, scale improvements, and other changes.
+
+*Release date:* February 24, 2021
+
+== New features
```
Comment (bergerhoffer) [inline release_notes/3056-release-notes.adoc:13]: I think we're supposed to add explicit anchors for all of these headings, both in modules and assemblies

File: release_notes/3056-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+=== Host scanning
+
+You can now identify vulnerabilities in core Kubernetes components and the container runtimes (Docker, CRI-O, runC, and containerd) you are using on your nodes.
+//See link:/docs/manage-vulnerabilities/#identify-vulnerabilities-in-nodes[Identify vulnerabilities in nodes] for more information.
+
+=== Network baseline updates
+
+The {product-title} discovers your deployments' network flows and creates a baseline based on the regular network activity in your clusters.
```
Comment (bergerhoffer) [inline release_notes/3056-release-notes.adoc:22]: Should "The" be removed at the beginning of the sentence?

File: release_notes/3056-release-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+You can now configure alerts and block network activity for connections that don't exist in the baseline.
```
Comment (bergerhoffer) [inline release_notes/3056-release-notes.adoc:23]: don't -> do not

File: release_notes/3056-release-notes.adoc [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+//For more information, see link:/docs/manage-network-policies/#use-network-baselining[Use Network baselining].
+
+== Important bug fixes
+
+* *ROX-3893*: Previously, if you used `--json` option with the `roxctl image check` command, it would always exit with code `0`.
```
Comment (bergerhoffer) [inline release_notes/3056-release-notes.adoc:28]: used -> used the

File: release_notes/3056-release-notes.adoc [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+
+* *ROX-6085*: Fixed some issues with the `roxctl` CLIs logging and debugging commands.
+* *ROX-6282*: Fixed the incorrect API description for `CountDeployments` and added missing descriptions for the `GetSecret`, `CountSecrets`, and `ListSecrets` endpoints.
+* *ROX-6303*: Previously, the *Deployment With Most Severe Violations* widget in the *Vulnerability Management* view did not change based on the selected namespace.
+
+* *ROX-6388*: Previously, the `sensor.sh` script would fail with errors when installing Sensor on {ocp} version 4.6 by using that script.
+
+* *ROX-6504*: Previously, the RHACS portal would forcefully log out users when they selected *Platform Configuration* -> *Access Control*, and they did not have appropriate permissions.
+
+* *ROX-6522*: Fixed an issue in the RHACS portal, where the component's location would not display if you were viewing details for a specific image.
```
Comment (bergerhoffer) [inline release_notes/3056-release-notes.adoc:38]: Comma could be removed. Check other where sentences too

File: release_notes/3056-release-notes.adoc [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+=== Resolved in version 3.0.56.1
+
+*Release date:* Mar 8, 2021
+
+* *ROX-6629*: Fixed an issue in the RHACS portal, where the *Compliance* view would sometimes display an error message.
+* *ROX-6708*: Previously, the Scanner would fail to scan images with OCI (Open Container Initiative) manifests.
+
+* *ROX-6699*: Previously, modifying the *Docker CIS 5.15: Ensure that the host's process namespace is not shared* policy would result in an error.
+
+* *ROX-6716*: Previously, if you have used Helm to install a Sensor, you could not enable automatic upgrades for that secured cluster.
```
Comment (bergerhoffer) [inline release_notes/3056-release-notes.adoc:52]: you have used -> you used

File: release_notes/3056-release-notes.adoc [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+You can now use the `SENSOR_HELM_NOT_HELM_MANAGED=true` environment variable to enable automatic upgrades for Helm managed secured clusters.
+
+== Known Issues
```
Comment (bergerhoffer) [inline release_notes/3056-release-notes.adoc:55]: Issues -> issues

File: release_notes/3056-release-notes.adoc [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+
+* CVE-2020-28928 discovered in alpine:3.13 is a false positive.
+This is actively being addressed.
+
+== Important system changes
+
+* Splunk alert events sent to HEC no longer include policy description, remediation, and rationale to allow for more violations within the HEC limit.
+* Added *Namespace* and *Node* to the minimal access specification for creating a new *Role*.
+* Added *Admission Control health status* to the link:/docs/configure-stackrox/use-health-dashboard/[health dashboard].
```
Comment (bergerhoffer) [inline release_notes/3056-release-notes.adoc:64]: Should this link be removed or something different?

File: release_notes/3056-release-notes.adoc [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+Navigate to *Platform Configuration* > *System Health* to view.
```
Comment (bergerhoffer) [inline release_notes/3056-release-notes.adoc:65]: `>` -> `->`

File: release_notes/3056-release-notes.adoc [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+This configuration is insecure, and Red Hat does not recommend using it.
+* If you are using the *Static configuration* option, you can now specify multiple PEM-encoded certificates for the `IdP Certificate(s) (PEM)` option.
+
+=== roxctl CLI
+
+* You can now use the new `--categories` option with the `roxctl image check` command and specify a comma-separated list of categories to only run policies that match the specified categories.
+* You can now use the new `--json-fail-on-policy-violations` option with the `roxctl image check` command.
+When you set this option to `true`, and there are policy violations, the command exits with a non-zero exit code.
+The default value for this option is `false`.
+* Red Hat plans to deprecate the `--json` option for the `roxctl image check` command in {product-title} version 3.0.59, use the `--json-fail-on-policy-violations` option instead.
```
Comment (bergerhoffer) [inline release_notes/3056-release-notes.adoc:101]: colon instead of the comma

File: release_notes/3057-release-notes.adoc [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+{product-title} scans all active (deployed) images every 4 hours and updates the image scan results to reflect the latest vulnerability definitions.
+You can now add inactive (undeployed) images for automatic scanning.
+//For more details, see link:/docs/manage-vulnerabilities/#scan-inactive-images[Scan inactive images].
+
+== Important bug fixes
+
+* *ROX-6085*: Previously, setting Central log level by using the `roxctl` CLI's `debug log` command would not work sometimes.
+You could also set unacceptable values for log level (for example, `Trace`), which did not affect the log level.
+
+* *ROX-6302*: The *Violations* view where loading too many violations would sometimes crash the RHACS portal page.
```
Comment (bergerhoffer) [inline release_notes/3057-release-notes.adoc:26]: Just checking, is "where" the right word? I'm having trouble understanding what the sentence is trying to say. Should it be "Loading too many violations on the **Violations** view would sometimes..."

File: release_notes/3057-release-notes.adoc [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+
+* *ROX-6627*: Previously, on {ocp}, when creating new builds, the {ocp} web console would show the error message  *admission webhook "policyeval.stackrox.io" does not support dry run*.
+{product-title} now includes dry run support for the admission controller webhook.
+
+* *ROX-6640*: The RHACS portal would not display the full description for RHSA CVEs in the *CVE* details view.
+* *ROX-6723*: Previously, if you were using the default CA certificate for Central, and you have configured an additional CA certificate for a Sensor, the {product-title} would overwrite Sensor's additional certificate.
```
Comment (bergerhoffer) [inline release_notes/3057-release-notes.adoc:32]: you have -> you

File: release_notes/3057-release-notes.adoc [hunk 14]
```diff
@@ -0,0 +1,101 @@
...
+
+* *ROX-6736*: Previously, sometimes the RHACS portal didn't show allowed connections between Sensor and non-isolated deployments in the *Network Graph*  view.
```
Comment (bergerhoffer) [inline release_notes/3057-release-notes.adoc:34]: didn't -> did not

File: release_notes/3057-release-notes.adoc [hunk 15]
```diff
@@ -0,0 +1,101 @@
...
+* *ROX-6832*: Previously, if you were using the RHEL base image, upgrading {product-title} to version 3.0.57.0 would fail.
+
+=== Resolved in version 3.0.57.2
+
+Release date: Mar 25, 2021
+
+* *ROX-6834*:  Previously, in {product-title} version 3.0.57.0, you could not install a Sensor using the RHACS portal because downloading the Sensor bundle would fail.
+
+* *ROX-6805*: Previously, if you were using {product-title} on {ocp}, the security context constraint link:https://bugzilla.redhat.com/show_bug.cgi?id=1824800[conflicted with the Authentication Operator].
+{product-title} now includes updated security context constraint which fixes this issue.
```
Comment (bergerhoffer) [inline release_notes/3057-release-notes.adoc:49]: now includes -> now includes an

File: release_notes/3057-release-notes.adoc [hunk 16]
```diff
@@ -0,0 +1,101 @@
...
+* 4 if you are deploying on {ocp} version `4.x`.
+
+When you do not specify this option, {product-title} generates deployment bundles in a compatibility mode that works on {ocp} version `3.11` and version `4.x`.
+However, if you are using {ocp} version `4.x`, Red Hat recommends that you specify this options as `4` to take advantage of additional features that are not available for earlier {ocp} versions.
+
+== Security updates
+
+{product-title} includes an updated Collector image which resolve the following fixable CVEs:
+
+* https://nvd.nist.gov/vuln/detail/CVE-2021-24031[(CVE-2021-24031]
```
Comment (bergerhoffer) [inline release_notes/3057-release-notes.adoc:75]: A random parenthesis here

File: release_notes/3058-release-notes.adoc [hunk 17]
```diff
@@ -0,0 +1,73 @@
...
+== Important bug fixes
+
+* *ROX-5397*, *ROX-6458*, and *ROX-6619*: Fixed a minor issue in the *Iptables Executed in Privileged Container* security policy, updated the remediation instructions in the *Curl in Image* security policy, and updated the *Kubernetes Dashboard Deployed* policy criteria.
+* *ROX-6497*: Previously, you could not use  OIDC Identity Provider with the Authorization Code Grant authentication flow, by using a client secret.
+The connection would fail with the *implicit grant not allowed for this client* error message.
+
+* *ROX-6626*: Previously, if you were using {product-title} on {ocp}, the *Network Graph* view would show too many connections and did not show Network baselines.
+
+* *ROX-6792*: Fixed an issue with the *inactive deployment* filter in the *Violations* view.
+* *ROX-6820*: Previously, {product-title} would not report CVE's in Distroless images under certain conditions.
```
Comment (bergerhoffer) [inline release_notes/3058-release-notes.adoc:22]: CVS's -> CVEs

File: release_notes/3058-release-notes.adoc [hunk 18]
```diff
@@ -0,0 +1,73 @@
...
+
+[role="_abstract"]
+{product-title} version 3.0.58 includes feature enhancements, bug fixes, scale improvements, and other changes.
+
+*Release date:* April 08, 2021
+
+== Important bug fixes
+
+* *ROX-5397*, *ROX-6458*, and *ROX-6619*: Fixed a minor issue in the *Iptables Executed in Privileged Container* security policy, updated the remediation instructions in the *Curl in Image* security policy, and updated the *Kubernetes Dashboard Deployed* policy criteria.
+* *ROX-6497*: Previously, you could not use  OIDC Identity Provider with the Authorization Code Grant authentication flow, by using a client secret.
```
Comment (bergerhoffer) [inline release_notes/3058-release-notes.adoc:16]: should Identity Provider be lowercase? and maybe an "an" after "use"?

File: release_notes/3058-release-notes.adoc [hunk 19]
```diff
@@ -0,0 +1,73 @@
...
+* *ROX-6820*: Previously, {product-title} would not report CVE's in Distroless images under certain conditions.
+
+* *ROX-6887*: Previously, the admission controller enforcement would not work for deploy-time policies if you were using `enforceOnUpdates`.
+
+=== Resolved in version 3.0.58.1
+
+Release date: Apr 20, 2021
+
+* *ROX-6959*: Previously, the {ocp} Cluster Version Operator was not correctly identified as an orchestrator component.
+We've fixed this issue.
```
Comment (bergerhoffer) [inline release_notes/3058-release-notes.adoc:31]: Rewrite all of the "we've" to get rid of "we"

File: release_notes/3058-release-notes.adoc [hunk 20]
```diff
@@ -0,0 +1,73 @@
...
++
```
Comment (bergerhoffer) [inline release_notes/3058-release-notes.adoc:32]: This + should be removed; the security updates heading isn't rendering properly

File: release_notes/3058-release-notes.adoc [hunk 21]
```diff
@@ -0,0 +1,73 @@
...
+== Security updates
+
+We've updated the Collector image to resolve the following fixable CVEs:
```
Comment (bergerhoffer) [inline release_notes/3058-release-notes.adoc:35]: here and next sentence, should it be singular CVE/RHSA

File: release_notes/3058-release-notes.adoc [hunk 22]
```diff
@@ -0,0 +1,73 @@
...
+
+We've updated all RHEL-based images to resolve the following fixable RHSAs:
+
+* https://access.redhat.com/errata/RHSA-2021:1206[RHSA-2021:1206]
+
+== Important system changes
+
+* Removed all licensing restrictions from {product-title}.
+* You can now enforce scheduling for the `scanner` and `scanner-db` deployments on specific nodes.
+* {product-title} includes a *Fixed by* column to the *Vulnerability Management* >  *All Entities* > *Components* view.
```
Comment (bergerhoffer) [inline release_notes/3058-release-notes.adoc:47]: `>` -> `->` for all of these?

File: release_notes/3059-release-notes.adoc [hunk 23]
```diff
@@ -0,0 +1,64 @@
...
+
+[role="_abstract"]
+{product-title} version 3.0.59 includes feature enhancements, bug fixes, scale improvements, and other changes.
+
+*Release date:* April 28, 2021
+
+== Important bug fixes
+
+* *ROX-6696*: {product-title} includes updates to AWS Security Hub integration to respect Amazon's rate limits on payload size to prevent errors.
+* *ROX-5299*, *ROX-6496*, *ROX-6856*, and *ROX-6718* *ROX-6472*:{product-title} includes updated Sensor which fixes multiple data race conditions, which resulted in errors.
```
Comment (bergerhoffer) [inline release_notes/3059-release-notes.adoc:16]: "and" is in the wrong spot

File: release_notes/3059-release-notes.adoc [hunk 24]
```diff
@@ -0,0 +1,64 @@
...
+
+=== Resolved in version 3.0.59.1
+
+Release date: May 4, 2021
+
+* *ROX-7053*: Previously, re-using a cluster name might cause autogenerated registry integrations to get duplicated.
```
Comment (bergerhoffer) [inline release_notes/3059-release-notes.adoc:22]: reusing

File: release_notes/3059-release-notes.adoc [hunk 25]
```diff
@@ -0,0 +1,64 @@
...
+
+=== Resolved in version 3.0.59.2
+
+Release date: May 13, 2021
+
+* *ROX-7154*: Previously, when using standalone pods with attached services customers would sometimes experience unstable deployments and crash loops.
```
Comment (bergerhoffer) [inline release_notes/3059-release-notes.adoc:28]: Comma after services?

File: release_notes/3059-release-notes.adoc [hunk 26]
```diff
@@ -0,0 +1,64 @@
...
+
+== Important system changes
+
+* *ROX-7016*: Product is now rebranded as {product-title}.
+* *ROX-6909*: The *Curl in image* and *Wget in image* policies and now disabled by default to reduce default violations.
```
Comment (bergerhoffer) [inline release_notes/3059-release-notes.adoc:33]: and now -> are now ?

File: release_notes/3060-release-notes.adoc [hunk 27]
```diff
@@ -0,0 +1,62 @@
...
+:context: 3060-release-notes
+
+toc::[]
+
+[role="_abstract"]
+{product-title} version 3.0.60 includes feature enhancements, bug fixes, scale improvements, and other changes.
+
+*Release date:* May 19, 2021
+
+== New Features
```
Comment (bergerhoffer) [inline release_notes/3060-release-notes.adoc:13]: Features -> features (a few other files too)

File: release_notes/3060-release-notes.adoc [hunk 28]
```diff
@@ -0,0 +1,62 @@
...
+=== Resolved in version 3.0.60.1
+
+Release date: May 26, 2021
+
+* *ROX-7253*: Previously, in version 3.0.60.0, scanner would fail to update vulnerability definitions and incorrectly report as healthy.
+
+
+== Important system changes
+
+* *ROX-7059*: {product-title} includes updated user interface to show resource summary counts after removing them in 3.0.58.0.
```
Comment (bergerhoffer) [inline release_notes/3060-release-notes.adoc:37]: includes -> includes an

File: release_notes/3061-release-notes.adoc [hunk 29]
```diff
@@ -0,0 +1,65 @@
...
+
+== Important bug fixes
+
+* *ROX-6991 and ROX-7058*: Previously, CSV exports of security risks were inconsistent with the RHACS user interface.
+
+* *ROX-7004*: Previously, CVE-2016-4074 was reported as a false positive when images contained the component `jq 1.6-r0` or `jq 1.6-r1`.
+
+* *ROX-7270*: Previously, under certain conditions, searched images would not correctly index and display.
+
+* *ROX-7276*: Previously, improper handling of very short-lived tokens caused the GitLab OIDC authentication provider to pre-maturely log users out.
```
Comment (bergerhoffer) [inline release_notes/3061-release-notes.adoc:29]: prematurely

File: release_notes/3061-release-notes.adoc [hunk 30]
```diff
@@ -0,0 +1,65 @@
...
+
+=== Resolved in version 3.0.61.1
+
+Release date: June 21, 2021
+
+* *ROX-7387*: Previously, in deployments using non-standard namespaces, admission controller failed to enforce or monitor deploy time policies by failing open on a certificate error.
```
Comment (bergerhoffer) [inline release_notes/3061-release-notes.adoc:35]: Is "failing open" the right words here?

File: release_notes/3061-release-notes.adoc [hunk 31]
```diff
@@ -0,0 +1,65 @@
...
+
+
+== Important system changes
+
+* *ROX-6639*: {product-title} includes a new default policy to flag fixable high or important severity vulnerabilities in images.
+* *ROX-7133*: {product-title} now calculates the Image risk using a score assigned to the severity rating of a vulnerability rather than the CVSS score.
```
Comment (bergerhoffer) [inline release_notes/3061-release-notes.adoc:41]: Just checking, does "Image" need to be capitalized?

**Comment by gaurav-nelson** [comment]
Thank you @bergerhoffer for reviewing. I've addressed you comments.

Please note that in the latest updated:
- I've added a new file `release_notes/3062-release-notes.adoc`.
- removed `release_notes/3056-release-notes.adoc` from the TOC as we plan to keep only the latest 6 release notes.

**Comment by bergerhoffer** [comment]
LGTM, thanks for the updates.

**Pull request merged**